.ci/build.sh

```diff
@@ -67,24 +67,31 @@ function build_it {
   export PATH="$HOME/.pyenv/bin:$PATH"
   export TERM=dumb
   export LC_ALL=en_US.UTF-8
-  export BENCHMARK_HOME=$WORKSPACE
+  export BENCHMARK_HOME="$GITHUB_WORKSPACE"
   export JAVA_PATH="/opt/hostedtoolcache/Java_Adopt_jdk"
   export JAVA_HOME="$JAVA_PATH/15.0.2-7/x64"
   export JAVA8_HOME="$JAVA_PATH/8.0.292-1/x64"
   export JAVA11_HOME="$JAVA_PATH/11.0.11-9/x64"
   export JAVA15_HOME="$JAVA_PATH/15.0.2-7/x64"
+  export JAVA16_HOME="$JAVA_PATH/16.0.2-7/x64"
 
   update_pyenv
   eval "$(pyenv init -)"
   # ensure pyenv shims are added to PATH, see https://github.com/pyenv/pyenv/issues/1906
   eval "$(pyenv init --path)"
   eval "$(pyenv virtualenv-init -)"
   pip install opensearch-benchmark
+  docker pull datadog/squid:latest
```

Comment:
Where does this get used?

Reply:
It is used in the integration tests for testing a proxy user. I am not entirely sure how that fits into the larger scope of OSB

https://github.com/opensearch-project/opensearch-benchmark/blob/main/it/proxy_test.py#L39